src/App.tsx

```diff
@@ -14,210 +31,74 @@ export const App: React.FC = () => {
     }
   }, []);
 
+  useEffect(() => {
+    if (user) {
+      try {
+        getTodos(user.id)
+          .then(setTodos);
+      } catch {
+        setError('Can\'t add todo');
```

Comment:
You don't add todo here. It would be better if you create another error message for this case